### PR TITLE
[TFKUBE-188] Review & edit user guide content

### PR DESCRIPTION
**Pull request by jkrawczyk-atlassian**
## Description

This PR contains the changes I made to the user guide content. The changes include linguistic, structural, and content improvements. Some of them may be significant, so a thorough verification of the work is required.

## Outstanding questions

I'm struggling to understand the content of the following help note in `INSTALLATION.md`:

```markdown
!!! help "How to run the product after installation?"    
    When the installation process finishes successfully, you can find some detailed information about the infrastructure on your console including the endpoint url (`product_urls`/`load_balancer_hostname`) to open the product on your browser and more. 
```

Can I ask for some clarification around this?

### CODE REVIEW

**Comment by nghazali** [comment]
When the installation process finishes successfully, you can find some detailed information about the infrastructure on your console including the endpoint url (`product_urls`/`load_balancer_hostname`) to open the product on your browser and more. 

Thanks Jacek, the help block you mentioned is about describing how user can run the product after installation finished successfully. 

This has slightly changed since we created this file. The new content should just point to 'product_url'. Here is the output of the installation:

```
product_urls = {
  "bamboo" = "http://a3707117c2a804c3786461c80fe4a29b-377155482.ap-northeast-1.elb.amazonaws.com"
}
```

As you see the url to the product is reported in product

**Comment by jkrawczyk-atlassian** [comment]
Ok, that makes sense now. I'll make an appropriate edit.

**Review by nghazali — APPROVED** [review]

File: docs/docs/userguide/INSTALLATION.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ If you want to use a custom configuration file to handle more than one environme
 ```
 
 !!! help "How to run the product after installation?"    
-    When the installation process finishes successfully, you can find some detailed information about the infrastructure on your console including the endpoint url (`product_urls`/`load_balancer_hostname`) to open the product on your browser and more.      
+    When the installation process finishes successfully, you can find some detailed information about the infrastructure in the console, including the `<product_url>`/`<load_balancer_hostname>` endpoint URL to launch the product in the browser and more.
```
Comment (nghazali) [inline docs/docs/userguide/INSTALLATION.md]: Please remove `/<load_balancer_hostname> endpoint URL` from this line.  Although we report `load_balancer_hostname` in the console, but this cannot be used to launch the product.

Comment (nghazali) [reply docs/docs/userguide/INSTALLATION.md]: Thanks, all good 👍

**Pull request merged**